pom.xml

```diff
@@ -48,6 +48,24 @@
             <version>1.18.12</version>
             <scope>provided</scope>
         </dependency>
+           <dependency>
```

Comment:
When adding modules that we depend on, we need to add these into `omod/src/main/resources/config.xml` so that OpenMRS loads this module in the appropriate order.

Reply:
oops i didnt know how i forgot this, i have resolved this thanks